### ISSUE TITLE
I sincerely ask the author so what version of KAPRE is it right for us to download

### ISSUE THREAD

**Issue by guoyuge** [seth814/Audio-Classification#75]
I sincerely ask the author so what version of KAPRE is it right for us to download

**Comment by seth814** [comment]
Is it not `kapre==0.3.4` ?
It's in requirements.txt.
I remember some months after I released this, he updated the kapre library.
The same functions exist, but he just moved the around in the later versions from what I remember.

**Comment by guoyuge** [comment]
Yeah, the function has moved, so notebook can't run a few of them, and he's changed the variables. I really can't rewrite it. I'm an idiot. I hope you have time to maintain a little, thank you very much




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "seth814/Audio-Classification"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2023年12月5日(星期二) 上午10:08
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [seth814/Audio-Classification] I sincerely ask the author so what version of KAPRE is it right for us to download (Issue #75)





 
Is it not kapre==0.3.4 ?
 It's in requirements.txt.
 I remember some months after I released this, he updated the kapre library.
 The same functions exist, but he just moved the around in the later versions from what I remember.

**Comment by seth814** [comment]
Just try `pip install kapre==0.3.4`
If not, it just looked like he moved the layers into kapre.composed.
https://github.com/keunwoochoi/kapre/blob/af4eb541d733cfed3ec57b10579307e9f5613359/kapre/composed.py
